### ISSUE TITLE
[Feature Request] Show Twitch Chat Synced with Playback of Video

### ISSUE THREAD

**Issue by Brendan-D15** [futo-org/grayjay-android#320]
When playing back a video on Twitch (that had been previously streamed), switching over the comments to "Platform" comments should playback any Twitch comments made during the stream.

The comments should play back in sync with the time that they were displayed during the stream.

**Comment by defiling9046** [comment]
@Zvonimir-FUTO Isn't this fixed with "VOD chat". I remember watching a Twitch VOD a while ago. I could be misunderstanding the feature request.

**Comment by Zvonimir-FUTO** [comment]
Yes this has been fixed with VOD Chat. Thank you for pointing it out.
